.eslintrc.js

```diff
@@ -8,7 +8,7 @@ module.exports = {
   rules: {
     'prettier/prettier': 'error',
     'func-names': ['warn', 'as-needed'],
-    'react/jsx-filename-extension': ['error', { extensions: ['.js'] }],
+    'react/jsx-filename-extension': ['error', { extensions: ['.js', '.jsx', 'ts', '.tsx'] }],
```

Comment:
As a question: is this fair game to have as its own commit?

I made this change as part of this PR, and as it's own individual commit as its distinct from the spirit of fixing the loading of the help page.

Is this better as it's own commit as part of this PR, or as a separate small PR.

Reply:
When it comes to small stuff like this, I generally like to have it all in the same PR if it's even tangentially related to the main point of the PR. If I spot a bug that's totally unrelated, but very small, I'll even add a commit to the PR to fix that separate bug and just note in the PR description "Contains an unrelated fix for such-and-such". In the end, PRs are transitory, it's the commit history that lasts.